devito/ops/operator.py

```diff
@@ -1,12 +1,63 @@
+from numpy import float32
+
+from devito import Dimension
 from devito.operator import OperatorRunnable
+from devito.ir.iet.utils import find_offloadable_trees
+from devito.ir.iet.nodes import Iteration, Call
+from devito.ir.iet.visitors import FindNodes
+from devito.ir.iet import Transformer, List, MetaCall
+from devito.types import Array
+from devito.symbolics import Macro
+
+from devito.ops.utils import namespace
+from devito.ops.transformer import opsit
 
 __all__ = ['Operator']
 
 
 class Operator(OperatorRunnable):
     """
-    A special :class:`OperatorCore` to JIT-compile and run operators through OPS.
+        A special :class:`OperatorCore` to JIT-compile and run operators through OPS.
     """
 
+    _default_includes = OperatorRunnable._default_includes + \
+        ['ops_seq.h', 'ops_lib_cpp.h']
+
     def _specialize_iet(self, iet, **kwargs):
-        raise NotImplementedError
```

Comment:
for this PR, this can stay ``NotImplementedError``. As I said, this should be a relatively small, self-contained PR

Reply:
If i do this, my tests will fail and wont be possible to run using DEVITO_BACKEND=ops.

I am not sure how can i remake the tests so i can test only the expression, i mean, i dont know how can i build the IterableTree manually to be used in the make_ops_ast.

Reply:
I do like this